bigchaindb/version.py

```diff
@@ -2,8 +2,8 @@
 # SPDX-License-Identifier: (Apache-2.0 AND CC-BY-4.0)
 # Code is Apache-2.0 and docs are CC-BY-4.0
 
-__version__ = '2.0.0b9'
-__short_version__ = '2.0b9'
+__version__ = '2.0.0'
```

Comment:
I think this would qualify as a 2.1 kind of change, at least, no?

Reply:
well, as this was a planned release (and TM upgrade part of the 2.0.0 but never added. I thought we put it to 2.0.0